src/cameras/PerspectiveCamera.js

```diff
@@ -99,6 +108,59 @@ class PerspectiveCamera extends Camera {
 
 	}
 
+	/*
+	 * Provides the topRight and bottomLeft corners of the camera's 2D frustum slice at a given distance.
+	 * Results are copied into minTarget and maxTarget input vars.
+	 */
+	getBounds( distance, minTarget, maxTarget ) {
+
+		_tempV3.set( - 1, - 1, 0.5 ).applyMatrix4( this.projectionMatrixInverse );
+		_tempV3.multiplyScalar( distance / Math.abs( _tempV3.z ) );
+		minTarget.x = _tempV3.x;
+		minTarget.y = _tempV3.y;
+
+		_tempV3.set( 1, 1, 0.5 ).applyMatrix4( this.projectionMatrixInverse );
+		_tempV3.multiplyScalar( distance / Math.abs( _tempV3.z ) );
+		maxTarget.x = _tempV3.x;
+		maxTarget.y = _tempV3.y;
+
+	}
+
+	/*
+	 * Calculates the height/width of the camera's frustum at a given distance.
+	 * returns a Vector2 where x is width and y is height.
+	 */
+	frustumDimensions( distance ) {
+
+		this.getBounds( distance, _minTarget, _maxTarget );
+		_dimensions.x = _maxTarget.x - _minTarget.x;
+		_dimensions.y = _maxTarget.y - _minTarget.y;
+
+		return _dimensions.clone();
+
+	}
+
+	/*
+	 * Calculates Vector3s of the frustum's corners at a given distance from the camera.
+	 * Returns an object with topLeft, topRight, bottomRight, and bottomLeft properties. Each property is a Vector3.
+	 */
+	frustumCorners( distance ) {
```

Comment:
This method belongs in `examples/jsm/utils/CameraUtils.js` IMO -- if at all.

Can you explain how these vertices are useful? It seems application-specific to me.